data_changes.md

```diff
@@ -0,0 +1,58 @@
+## Changes to Imported Data ##
+
+Generally, data will be imported as-is from historical data or newly given data
+with a few exceptions:
+
+- _IMPORT TYPE_: Broker, USASpending
```

Comment:
My .02 on the `IMPORT TYPE` is to leave it out unless someone asks for it. If there's something that applies to one type of load but not another, we can also note it inline (which you've already done in places like line 31).